### DISCUSSION TITLE
Visual Studio Code

### DISCUSSION THREAD

**Discussion by kspeul** [hlorenzi/customasm#164] [Q&A]
I tried to use the customasm extension in VSC with some code I had compiled using the command line version.  I have #included some files in my main file and it doesn't see them in the directory.  VSC compiler gives me an error of "file not found".  The code compiles fine in the command line version of customasm.

**Comment by hlorenzi** [comment]
Hmm, what do you mean the "VSC compiler"? Do you mean you're using the app through VSCode's integrated terminal?
The `customasm-vscode` extension itself currently doesn't attempt to compile the program, providing just very basic syntax highlighting. Could you provide the failing source-code, or perhaps even a screenshot with more context, showing VSCode's terminal?

**Comment by kspeul** [comment]
[VisualStudioCodeCompile.odp](https://github.com/hlorenzi/customasm/files/11107626/VisualStudioCodeCompile.odp)

**Reply by kspeul** [reply]
I just noticed that the extension is not yours but Joseph Abbey.  Yours is the highlighting.  Sorry about that.  I will try and get a hold of him.

**Reply by hlorenzi** [reply]
I see! I've been preparing some things now, and we might have a fully-fledged official extension in the future.
